### PR TITLE
Installing in Azure

### PR DESCRIPTION
**Pull request by mendezcirera**
Additional and more detailed content for deploying 128T in Azure. This new content extends the previous short installation guide

### CODE REVIEW

File: docs/intro_installation_azure.md [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+![Plans](/img/platforms_azure_plans.png)
+
+Lastly click on the "Launch" link of the template that better suits your needs.
+
+Answer the following 3 questions to launch the deployment of a 128T Conductor (additional information [here](#launch-the-template)):
+* What name do you want to give it? Provide it in the "Instance Name" field (for example: 128TConductor).
+* Where do you want to deploy it? Provide the location where the VNet exists in the "Location" field (for example: eastus. All available locations [here](https://azure.microsoft.com/en-us/global-infrastructure/locations)), the name of the VNet in the "Virtual Network Name" field (for example: 128T-VNet) and the name of a subnet in the "Control Subnet Name" field (for example: default).
+* Who is going to be the administrator? Provide an username (for example: t128) in the "Admin Username" field and the content of your public SSH key in the "Admin Public Key Data" field respectively.
+
+Agree to the terms of use and conditions and lastly click on the "Purchase" button to launch the deployment.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:48]: They have to agree again? Didn't they agree on line 37?

File: docs/intro_installation_azure.md [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+
+![Plans](/img/platforms_azure_deployment_complete.png)
+
+Once the deployment completes information of the newly 128T Conductor deployment is provided in the Outputs tab placed on the left hand side. Click on the HTTPs URL to login to the 128T Conductor GUI (if the web browser used is Chrome and it does not trust the connection to the 128T Conductor GUI due to its self-signed certificate type: thisisunsafe). The credentials are "admin" for username and the name of the VM for the password. To login to the VM via SSH use the username and the SSH public key provided in the template.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:52]: comma after "completes"

File: docs/intro_installation_azure.md [hunk 3]
```diff
@@ -0,0 +1,747 @@
...
+
+:::important
+Be sure to change the password that conforms to your business' password requirements and criteria.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:56]: remove trailing "important".  You just need a trailing :::

File: docs/intro_installation_azure.md [hunk 4]
```diff
@@ -0,0 +1,747 @@
...
+
+![Plans](/img/platforms_azure_programmatically.png)
+
+Click on the tab "Plans" as shown in the following picture:
+
+![Plans](/img/platforms_azure_plans.png)
+
+Lastly copy to the clipboard the URL of the template located in the field "URL" that better suits your needs.
+
+Create the parameters file, accept the terms of use and conditions and lastly launch the deployment with the corresponding Azure CLI or PowerShell commands making use of the URL of the template identified previously. For additional information please click [here](#launch-the-template).
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:70]: same comment as above.  Agree twice?

File: docs/intro_installation_azure.md [hunk 5]
```diff
@@ -0,0 +1,747 @@
...
+
+Once the deployment completes information of the newly 128T Conductor deployment is provided in the Outputs section. Click on the HTTPs URL to login to the 128T Conductor GUI (if the web browser used is Chrome and it does not trust the connection to the 128T Conductor GUI due to its self-signed certificate type: thisisunsafe). The credentials are "admin" for username and the password is the name of the VM. To login to the VM via SSH use the username and the SSH public key provided in the template.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:72]: comma after "completes"

File: docs/intro_installation_azure.md [hunk 6]
```diff
@@ -0,0 +1,747 @@
...
+
+:::important
+Be sure to change the password that conforms to your business' password requirements and criteria.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:76]: remove trailing "important". You just need a trailing :::

File: docs/intro_installation_azure.md [hunk 7]
```diff
@@ -0,0 +1,747 @@
...
+* The existing VNet should be segmented with at least three subnets. The role of each subnet is described below:
+  * Public subnet. The expectation is that this subnet provides connectivity to enable communication with external/remote 128T Router peers.
+  * Private subnet. The expectation is that this subnet provides connectivity to internal workloads within the cloud.
+  * Management subnet. The expectation is that this subnet meets the following capabilities:
+    * This subnet should be reachable via SSH for administration purposes.
+    * The interface of the 128T Conductor that is going to manage this router must be reachable from this subnet.
+
+:::important
+Please note that deploying 128 Technology Session Smart Routers without a valid certificate will be limited to deployments within the cloud only. If your use case requires the deployment of a 128T Router on your premises as well please contact 128 Technology directly [here](https://www.128technology.com/get-started).
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:94]: remove trailing "important". You just need a trailing :::

File: docs/intro_installation_azure.md [hunk 8]
```diff
@@ -0,0 +1,747 @@
...
+* What name do you want to give it? Provide it in the "Instance Name" field (for example: 128TRouter).
+* Where do you want to deploy it? Provide the location where the VNet exists in the "Location" field (for example: eastus. All available locations [here](https://azure.microsoft.com/en-us/global-infrastructure/locations)), the name of the VNet in the "Virtual Network Name" field (for example: 128T-VNet), the name of the availability set in the "Availability Set Name" field (for example: 128TRouterSet) and the name of the public, private and management subnets in the "Public Subnet Name", "Private Subnet Name" and "Management Subnet Name" fields respectively (for example: wan, lan and default).
+* Which 128T Conductor is going to manage it? Provide the IP address of the primary node of Conductor in the "Conductor Primary Control IP" field, and only if the Conductor is highly available then provide the IP address of the secondary node of Conductor in the "Conductor Secondary Control IP" field. Please check the public IP address assigned to the 128 Technology Conductor deployed in the previous section.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:122]: for each of these bulleted sections, it would be good to add a line break <br/>

File: docs/intro_installation_azure.md [hunk 9]
```diff
@@ -0,0 +1,747 @@
...
+* Who is going to be the administrator? Provide an username (for example: t128) and the content of your public SSH key in the "Admin Username" and "Admin Public Key Data" fields respectively.
+
+Agree to the terms of use and conditions and lastly click on the "Purchase" button to launch the deployment.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:125]: agree again?

File: docs/intro_installation_azure.md [hunk 10]
```diff
@@ -0,0 +1,747 @@
...
+
+![Plans](/img/platforms_azure_programmatically.png)
+
+Click on the tab "Plans" as shown in the following picture:
+
+![Plans](/img/platforms_azure_plans.png)
+
+Lastly copy to the clipboard the URL of the template located in the field "URL" that better suits your needs.
+
+Create the parameters file, accept the terms of use and conditions and lastly launch the deployment with the corresponding Azure CLI or PowerShell commands making use of the URL of the template identified previously. For additional information please click [here](#launch-the-template).
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:145]: agree again?

File: docs/intro_installation_azure.md [hunk 11]
```diff
@@ -0,0 +1,747 @@
...
+
+Click on the desired template, a new tab will be opened in the browser which will redirect you to its form. 
+
+:::important
+As an additional note and only applicable when the chosen image is the 128T Networking Platform, please be aware of the following conditions before using any of its templates:
+* Applicable when deploying either a 128 Technology Conductor or a 128T Session Smart Router using the 128T Networking Platform image only:
+  * The management network must allow outbound access to the Internet so that the 128T installer can download the 128T software from the 128T YUM repositories available on the Internet.
+* Applicable when deploying a 128T Session Smart Router using the 128T Networking Platform image only:
+  * If there is an existing 128T Conductor in the network and the intent is to perform a non interactive installation of a 128T Router (for example a ZTP installation) please make sure the certificate provided by 128 Technology has been imported and loaded in 128T Conductor before launching this template.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:351]: do not need trailing "important"

File: docs/intro_installation_azure.md [hunk 12]
```diff
@@ -0,0 +1,747 @@
...
+* Public IP address for administration purposes.
+* HTTPs URL to login to the 128T Conductor GUI. Please continue to the end of this section below for more information regarding the credentials to login.
+* SSH command to login to the Linux VM. Please continue to the end of this section below for more information regarding the credentials to login.
+
+:::important
+When logging to the Linux instance via SSH make use of the username specified in the "Admin Username" field and the corresponding private key specified in the "Admin Public Key Data" field. 
+When logging to the 128T application via CLI or HTTPs the username is "admin" and the password:
+* For Hourly and Private images the password is the name of the VM.
+* For BYOL images the password is the one that was specified during the interactive initialization process.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:398]: remove trailing "important"

File: docs/intro_installation_azure.md [hunk 13]
```diff
@@ -0,0 +1,747 @@
...
+| Admin Username       | Fill out the field Admin Username with the desired username to login to the VM (Linux) via SSH.                                                                              |
+| Admin Public Key Data| Paste in the field Admin Public Key Data the SSH public key to be used to authenticate with the VM (Linux) instance via SSH. The key needs to be at least 2048-bit and in ssh-rsa format. Please find the following an example of a valid key next (To reduce the length of the key in this example multiple character have been replaced by three dots): ```ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDHwB1Qe1KndGqKuT3F...GumfdHfdasy8N0kncMtp2wtkqoLsRWdJ4/WKaZBOrPd4Q== admin@Admin-MacBook-Pro.local```. For more information about creating ssh keys, see [Create SSH keys on Linux and Mac for Linux VMs in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/mac-create-ssh-keys).                                                                                                         |
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:543]: I hope this is a fake key

File: docs/intro_installation_azure.md [hunk 14]
```diff
@@ -0,0 +1,747 @@
...
+
+If no IP address/es of a Conductor were provided an unmanaged router will be deployed, initialized, and ready to be configured, in this case proceed to login to the router via SSH.
+
+:::important
+When logging to the Linux instance via SSH make use of the username specified in the "Admin Username" field and the corresponding private key specified in the "Admin Public Key Data" field.
+When logging to the 128T application via CLI or HTTPs the username is "admin" and the password will be:
+* For Hourly and Private images the password is the name of the VM.
+* For BYOL images the password is the one that was specified during the interactive initialization process.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:567]: remove trailing "important"

File: docs/intro_installation_azure.md [hunk 15]
```diff
@@ -0,0 +1,747 @@
...
+-TemplateParameterFile ./router_byol.parameters.json
+```
+
+### Certificate SAS URL
+
+A SAS URL can be used to perform a deployment of a 128 Technolody Conductor using the 128T Networking Platform image. It allows for a non interactive installation of the 128T Conductor fully automated and on your behalf as part of the deployment process. This section describes how a SAS URL for your 128 Technology certificate is created.
+
+:::important
+This section only applies when deploying a 128 Technology Conductor using the 128T Networking Platform image and the "Standalone Conductor" template.
+:::important
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:729]: remove trailing "important"

File: docs/intro_installation_azure.md (hunk 1 above)
Comment (mendezcirera) [reply docs/intro_installation_azure.md:48]: Good point. Yes, you have to agree again, but let me actually clarify: the first time is to accept the terms of the image, the second time is to accept the terms of the deployment via the template. Let me make it more clear in the doc

File: docs/intro_installation_azure.md (hunk 4 above)
Comment (mendezcirera) [reply docs/intro_installation_azure.md:70]: Thanks! I will make it more clear throughout the doc

File: docs/intro_installation_azure.md (hunk 13 above)
Comment (mendezcirera) [reply docs/intro_installation_azure.md:543]: It is a public key, I cut a third of it.

File: docs/intro_installation_azure.md (hunk 8 above)
Comment (mendezcirera) [reply docs/intro_installation_azure.md:122]: Would it be like below? break line between each bullet (mm I think the space between lines is too large?)

### Answer the following 4 questions to launch the deployment of a 128T Session Smart Router (additional information [here](#launch-the-template)):
* What name do you want to give it? Provide it in the "Instance Name" field (for example: 128TRouter).

* Where do you want to deploy it? Provide the location where the VNet exists in the "Location" field (for example: eastus. All available locations [here](https://azure.microsoft.com/en-us/global-infrastructure/locations)), the name of the VNet in the "Virtual Network Name" field (for example: 128T-VNet), the name of the availability set in the "Availability Set Name" field (for example: 128TRouterSet) and the name of the public, private and management subnets in the "Public Subnet Name", "Private Subnet Name" and "Management Subnet Name" fields respectively (for example: wan, lan and default).

* Which 128T Conductor is going.........

File: docs/intro_installation_azure.md [hunk 16]
```diff
@@ -206,7 +211,7 @@ Once in the Marketplace type "128 Technology" in the search bar within the Marke
 
 Next click on the "128 Technology Conductor" offering:
 
-![128 Technology Conductor Hourly](/img/platforms_azure_marketplace_conductor_hourly.png)
+<img src="/img/platforms_azure_marketplace_conductor_hourly.png" alt="128 Technology Conductor Hourly" width="220" height="250" />
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:214]: Why did you change this to an embedded image instead of letting the MD->HTML conversion do its job?

Comment (mendezcirera) [reply docs/intro_installation_azure.md:214]: For some reason some screenshots were larger than others, however I took all of them the same way...not sure why this is happening. 
I did try to resize them using MD but it always resulted in error.
With the HTML img tag I can define width and height to resize the images.
